test/integration.test.js

```diff
@@ -55,4 +55,38 @@ describe('Integration testing generateFromFile() to make sure the result of the
     const file = await readFile(path.join(outputDir, testOutputFile), 'utf8');
     expect(file).toMatchSnapshot();
   });
+
+  it('generates without transpilation when compile flag is false', async () => {
+    const outputDir = generateFolderName();
+    const generator = new Generator('@asyncapi/html-template@2.3.0', outputDir, {
+      forceWrite: true,
+      compile: false,
+      debug: true,
+    });
+  
+    const logSpy = jest.spyOn(console, 'log').mockImplementation(() => {});
```

Comment:
to be consistent in project, for spying on console, please follow the same approach we have in `generator.test.js`

log.debug = jest.fn();
expect(log.debug).toHaveBeenCalledWith("redacted for brevity));